README.md

```diff
@@ -39,17 +39,22 @@ CCX Notification Writer service
 The main task for this service is to listen to configured Kafka topic, consume
 all messages from such topic, and write OCP results (in JSON format) with
 additional information (like organization ID, cluster name, Kafka offset etc.)
-into database table named `new_reports`. Multiple reports can be consumed and
+into a database table named `new_reports`. Multiple reports can be consumed and
 written into the database for the same cluster, because the primary (compound)
 key for `new_reports` table is set to the combination `(org_id, cluster,
 updated_at)`. When some message does not conform to expected schema (for
-example if `org_id` is missing), such message is dropped and the error is
-stored into log.
+example if `org_id` is missing for any reason), such message is dropped and the
+error message with all relevant information about the issue is stored into the
+log. Messages are expected to contain `report` body represented as JSON.
+This body is shrink before it's stored into database so the database
```

Comment:
shrunk

Reply:
yeah my Czenglish ;) fixed and thank you!